Area51/Sources/SettingsViewController.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  SettingsViewController.swift
+//  Area51
+//
+//  Created by Felix Förtsch on 15.01.19.
+//
+
+import UIKit
+import SafariServices
+
+class SettingsViewController: UITableViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        switch indexPath.row {
+        case 0:
+            let changelogURL = "https://github.com/kgellci/Area51/blob/master/CHANGELOG.md"
+            guard let url = URL.init(string: changelogURL) else {
```

Comment:
Can just do URL(string:

Reply:
Oh, cool. Is the `guard let` with `fatalError()` correct here?

Reply:
Not really needed but can be refactored later. I will add an extension for non optional URL from string to make this case easier.